<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_workingMode">
           <property name="text">
            <string>Working mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_typeOfData">
           <property name="text">
            <string>Type of Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_inputMethod">
           <property name="text">
            <string>Input method</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_sortingMethod">
           <property name="text">
            <string>Sorting method</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QComboBox" name="comboBox_workingMode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Simple sorting</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Studing mode</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Research mode</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_typeOfData">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Integer</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Float</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Double</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Char</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>String</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_inputMethod">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Random Data</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>User Data</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_sortingMethod">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Bubble Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bubble Sort modificated</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Selection Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Insertion Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shelle Sort</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Merge Sort</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>11</y>
      <width>420</width>
      <height>167</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="Widget_DataPath" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_dataPath">
              <property name="text">
               <string>Data Path</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>50</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_dataPath"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Widget_RandomData" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_amountOfData">
              <property name="text">
               <string>Amount of Data</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBox_amountOfData">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" rowspan="2">
             <widget class="QLabel" name="label_range">
              <property name="text">
               <string>Range</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_rangeMin">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBox_rangeMax">
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="Widget_SortingMethod" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="cB_BubbleSort">
            <property name="text">
             <string>Bubble Sort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cB_BubbleSort_m">
            <property name="text">
             <string>Bubble Sort Modificated</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cB_InsertionSort">
            <property name="text">
             <string>Insertion Sort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cB_SelectionSort">
            <property name="text">
             <string>Selection Sort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cB_ShellSort">
            <property name="text">
             <string>Shell Sort</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cB_MergeSort">
            <property name="text">
             <string>Merge Sort</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>140</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QCheckBox" name="checkBox_pResult">
       <property name="text">
        <string>Print the result</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_pTime">
       <property name="text">
        <string>Print the time of sorting</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>261</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Sort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Sort</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>520</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
